tests/testthat/test-rust-sitrep.R

```diff
@@ -129,3 +129,27 @@ test_that("Required target is not available", {
   })
   expect_snapshot(rust_sitrep())
 })
+
+test_that("Detects host when default toolchain is not set (MacOS)", {
+  skip_if_not(get_os() == "osx")
```

Comment:
I would want to be careful mixing prod code with test code. Here it also checks only for `osx`.